rakelib/rhel_alts.rake

```diff
@@ -3,22 +3,22 @@ task :rhel_alts do
   Dir[File.join(__dir__, '..', 'facts', '*', 'centos-*-x86_64.facts')].each do |f|
     centos_factset = File.read(f)
 
-    rhel_path = f.gsub(%r{centos-}, 'redhat-')
+    rhel_path = f.gsub(/centos-/, 'redhat-')
     File.open(rhel_path, 'w') do |fd|
-      fd.puts centos_factset.gsub(%r{CentOS}, 'RedHat')
+      fd.puts centos_factset.gsub(/CentOS/, 'RedHat')
     end
```

Comment:
Probably unrelated for this PR, but should this use `File.write(rhel_path, centos_factset.gsub(/CentOS/, 'RedHat'))`

Reply:
done